include/boost/pfr/detail/fields_count.hpp

```diff
@@ -295,7 +297,13 @@ constexpr std::size_t fields_count() noexcept {
 //    );
 //#endif
 
+#if defined(_MSC_VER) && (_MSC_VER <= 1920)
+    // Workaround for msvc compilers. Versions <= 1920 have a limit of max 1024 elements in template parameter pack
+    constexpr std::size_t max_fields_count = boost::static_unsigned_min<sizeof(type) * CHAR_BIT, 1024>::value;
```

Comment:
```suggestion
    constexpr std::size_t max_fields_count = (sizeof(type) * CHAR_BIT >= 1024 ? 1024 : sizeof(type) * CHAR_BIT);
```
No very elegant, but works

Reply:
ok